src/main/frontend/dicts.cljs

```diff
@@ -284,9 +284,11 @@
         :sync-from-local-files "Refresh"
         :sync-from-local-files-detail "Import changes from local files"
         :unlink "unlink"
+        :search/search-header "Search"
+        :search/quick-capture "Quick Capture"
         :search (if config/publishing?
                   "Search"
-                  "Search or create page")
+                  "Type a note, create a page, or search your graph")
```

Comment:
When we change an existing dictionary entry, how do we make changes to all the other languages? I would be inclined to delete all the other :search language entries in this PR so that translators know to redo the translation